tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/firebase/MemoryFirebaseSubscriptionRepositoryModule.java

```diff
@@ -0,0 +1,19 @@
+package com.linagora.tmail.james.jmap.firebase;
+
+import org.apache.james.user.api.DeleteUserDataTaskStep;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+import com.google.inject.multibindings.Multibinder;
+
+public class MemoryFirebaseSubscriptionRepositoryModule extends AbstractModule {
```

Comment:
I feel like memory guice modules...

 - Are not realy related to the transport/protocol layer
 - Are only used by apps/memory-app
 
 Why not relocate them to apps/memory-app

Reply:
Nice idea!